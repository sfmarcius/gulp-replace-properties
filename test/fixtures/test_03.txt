Hello ABC javascript!
Hello ABC java!
Hello ABC node!
