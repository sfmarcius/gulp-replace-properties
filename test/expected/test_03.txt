Hello 123 javascript!
Hello 123 java!
Hello 123 node!
